Source/Order.swift

```diff
@@ -159,4 +152,93 @@ Represents an order placed by the user to be tracked using `ButtonMerchant.track
             case advertisingId = "advertising_id"
         }
     }
+
+    /**
+     Represents a line item in the order.
+     */
+    final public class LineItem: NSObject, Codable {
+
+        /**
+         The unique identifier for this line item,
+         within the scope of this order. (required).
+         */
+        let identifier: String
+
+        /**
+         The total price of all items bought in a particular line item (required).
+         (e.g. if 3 bananas were purchased for $3.00 each, total would be 900).
+         */
+        let total: Int64
+
+        /**
+         The number of unique units represented by this line item.
+         */
+        let quantity: Int
```

Comment:
add `(required).` to be consistent with the other properties

Reply:
Quantity is optional so we should add a ? and default to nil

Reply:
@wessmith quantity is not required

Reply:
Was going by the doc

<img width="473" alt="Screen Shot 2019-06-24 at 2 52 29 PM" src="https://user-images.githubusercontent.com/1057077/60044133-b99bdc80-968f-11e9-9ff2-fc12cf21a46d.png">

Reply:
<img width="325" alt="Screen Shot 2019-06-24 at 2 54 04 PM" src="https://user-images.githubusercontent.com/1057077/60044241-ee0f9880-968f-11e9-9563-2453a2960579.png">

Reply:
I'll update the doc accordingly